app/setup/routes.py

```diff
@@ -105,8 +109,9 @@ def get_installation_script(id: int):
         text = text.replace("$DEVICE_ID", str(id))
 
         return text
-    
+
 @setup_bp.route('/install_ip')
+@login_required
```

Comment:
Same here, 
this getter returns the installation script for devices and should be free